app/src/main/res/values/strings.xml

```diff
@@ -208,6 +208,7 @@
     <string name="sort_method_artist">Artist</string>
     <string name="sort_method_album">Album</string>
     <string name="sort_method_year">Year</string>
+    <string name="sort_method_date_added">Date Added</string>
```

Comment:
I'm not sure using a capitalized word in the middle of the "sentence" is a right idea.

```suggestion
    <string name="sort_method_date_added">Date added</string>
```

Reply:
I prefer title casing in most areas, I was actually going to go through the strings at some point to find all the inconsistent strings in places like this.